pandas/tests/io/test_sql.py

```diff
@@ -823,6 +823,21 @@ def test_get_schema_keys(self):
         constraint_sentence = 'CONSTRAINT test_pk PRIMARY KEY ("A", "B")'
         assert constraint_sentence in create_sql
 
+    @pytest.mark.parametrize("index_arg, expected", [
```

Comment:
I think it would be better if you simply parametrized on True and False and made "expected" the actual expected statement